encoders/nlp/UniversalSentenceEncoder/requirements.txt

```diff
@@ -1,4 +1,4 @@
-protobuf>=3.9.2
-grpcio>=1.24.3
-tensorflow
-tensorflow_hub
\ No newline at end of file
+protobuf==3.14.0
+grpcio==1.34.0
```

Comment:
Why are they sometimes removed and sometimes bumped? Shouldn't they come from core directly?

Reply:
when they are removed is because some tests were failing and dig deeper to fix it.

Reply:
trying to remove noew